Client/mods/deathmatch/logic/CClientModelManager.cpp

```diff
@@ -72,6 +72,22 @@ CClientModel* CClientModelManager::FindModelByID(int iModelID)
     return nullptr;
 }
 
+std::vector<CClientModel*> CClientModelManager::GetModelsByType(const eClientModelType type, const unsigned int minModelID)
+{
+    std::vector<CClientModel*> found;
+    found.reserve(16); // just pre-allocate some space, because we know there might be a few models
```

Comment:
16 seems very low for a reserve. Ideally we'd just use the actual count of gta_sa ped models here

Reply:
Its not that low actually. I could allocate 32, but more than that is unnecessary, because only custom models are stored here, the GTA ones arent.
But I could reserve 512, I mean thats only a few KB, so, doesnt really matter, and the value is RVOd anyways, so in theory there should be no copy of memory at all.

Reply:
RVO doesn't neccesarily apply due to `push_back`. But yeah, you're right here. I was thinking about `CClientModelManager` containing all (including GTA) models. 16 is probably fine here.

Reply:
Doesn't the first push_back effectively do a reserve anyway?